src/npm-utils.js

```diff
@@ -56,6 +58,20 @@ var execViewCommand = function(args) {
   return deferred.promise;
 };
 
+var execPackCommand = function(args) {
```

Comment:
Would be nice to put some jsdoc on this method (params, return type) ;) probably a link to npm docs about what `args` is